File: TH buổi 3 lap 6+7+8/LTMB_TH3_Lap8/app/src/main/java/com/example/ltmb_th3_lap8/MainActivity.java
package com.example.ltmb_th3_lap8;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button btnBasic, btnCustomList, btnGrid;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnBasic = findViewById(R.id.btnBasic);
        btnCustomList = findViewById(R.id.btnCustomList);
        btnGrid = findViewById(R.id.btnGrid);

        btnBasic.setOnClickListener(v ->
                startActivity(new Intent(this, BasicListActivity.class)));

        btnCustomList.setOnClickListener(v ->
                startActivity(new Intent(this, CustomListActivity.class)));

        btnGrid.setOnClickListener(v ->
                startActivity(new Intent(this, CustomGridActivity.class)));
        Button btnCrud = findViewById(R.id.btnCrud);
        btnCrud.setOnClickListener(v ->
                startActivity(new Intent(this, CrudFoodActivity.class)));

    }
}

File: Lab1_BasicApp/app/src/main/java/com/example/lab1_basicapp/MainActivity.kt
package com.example.lab1_basicapp

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private lateinit var btnOpenDice: Button
    private lateinit var btnOpenCallSms: Button
    private lateinit var btnOpenLoading: Button
    private lateinit var btnOpenSplash: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnOpenDice = findViewById(R.id.btnOpenDice)
        btnOpenCallSms = findViewById(R.id.btnOpenCallSms)
        btnOpenLoading = findViewById(R.id.btnOpenLoading)
        btnOpenSplash = findViewById(R.id.btnOpenSplash)

        //  Bài 1 - Tung xúc xắc
        btnOpenDice.setOnClickListener {
            startActivity(Intent(this, DiceActivity::class.java))
        }

        //  Bài 2 - Gọi điện và nhắn tin
        btnOpenCallSms.setOnClickListener {
            startActivity(Intent(this, CallSmsActivity::class.java))
        }

        //  Bài 3 - Loading UI
        btnOpenLoading.setOnClickListener {
            startActivity(Intent(this, LoadingActivity::class.java))
        }

        //  Mở màn hình Splash
        btnOpenSplash.setOnClickListener {
            startActivity(Intent(this, SplashActivity::class.java))
        }
    }
}
